uta_tools/schemas.py

```diff
@@ -11,6 +11,14 @@
 from uta_tools.version import __version__
 
 
+class TranscriptPriorityLabel(str, Enum):
+    """Create Enum for Transcript Priority labels"""
+
+    MANESelect: Literal["mane_select"] = "mane_select"
+    MANEPlusClinical: Literal["mane_plus_clinical"] = "mane_plus_clinical"
+    LongestCompatibleRemaining: Literal["longest_compatible_remaining"] = "longest_compatible_remaining"  # noqa: E501
```

Comment:
I'm not sure these are Literals -- if you run the type checking it produces some weird errors (eg that TranscriptPriorityLabel.LongestCompatibleRemaining doesn't have a 'value' method)

Reply:
Hmm I am unable to reproduce when running mypy. I think this may be a larger issue about having a hook to do type checking.

Reply:
Wait I don't even know why I put Literals here for an enum. I guess this is what happens when you work at night haha. I'm going to remove!